### PR TITLE
CMS: add Hubspot form to Benefits page template for organizations link

### LINKED ISSUE #4758: CMS: add Hubspot form to "Benefits" page template
As a MicroMasters B2B salesperson, I'd like to add an optional contact form to the "Benefits" page type.  (The "Benefits" page template is actually a generic page template, but we unfortunately named it "Benefits" after the first use case) 

#### Designs and Mockups

TBD. Please make a draft first and then we can adjust the styling. 

#### Acceptance Criteria:

- [ ] Do a little research to determine the best way to add embedded HTML into a Wagtail template
- [ ] Update this ticket with description of how the form can be added
- [ ] Allow the CMS user to optionally add a custom form like the following to any "Benefits" page type
```
<script charset="utf-8" type="text/javascript" src="//js.hsforms.net/forms/shell.js"></script>
<script>
  hbspt.forms.create({
	portalId: "8677455",
	formId: "1b63db1a-eb3a-45d6-82f1-c4b8f01835dc"
});
</script>
```

#### Related issues

- the home page redesign in #4736 includes a link to an "enterprises and institutions" page which would use this feature

### PR DESCRIPTION
**Pull request by umarmughal824**
#### Pre-Flight checklist

- [x] Testing
  - [x] Code is tested
  - [x] Changes have been manually tested

#### What are the relevant tickets?
fixes #4758 

#### What's this PR do?
CMS: add Hubspot form to Benefits page template for `/orgainzations/` link

#### How should this be manually tested?
Just create the benefits page with organizations slug

#### Screenshots (if appropriate)

**Organizations Page**

Screenshot 2021-01-25 at 14 49 45

**Benefits Page**

Screenshot 2021-01-25 at 14 49 53

### CODE REVIEW

**Comment by pdpinch** [comment]
Looking good. For the other sites we pull the Hubspot ID into a setting, which gives us the ability to differentiate between RC and prod and also to control the roll out. Can we do that here as well?

**Comment by umarmughal824** [comment]
@annagav @pdpinch that's ready for review now.

**Comment by pdpinch** [comment]
Can you merge this into your branch for the home page changes, so I can see it on the same PR deployment?

**Comment by umarmughal824** [comment]
> Can you merge this into your branch for the home page changes, so I can see it on the same PR deployment?

sure

**Review by annagav — APPROVED** [review]
👍

File: app.json [hunk 1]
```diff
@@ -124,6 +124,14 @@
       "description": "Value provided by Heroku containing the parent app name (eg micromasters-ci for a PR build)",
       "required": true
     },
+    "HUBSPOT_NEW_COURSES_FORM_GUID": {
+      "description": "Form guid over hub spot for new courses email subscription form.",
+      "required": false
+    },
```
Comment (pdpinch) [inline app.json:130]:
```suggestion
    "HUBSPOT_ORGANIZATIONS_FORM_GUID": {
      "description": "Hubspot guid for Organizations contact form on /organizations/ page",
      "required": false
    },
```

We can make this more specific to the use case on MicroMasters

**Comment by pdpinch** [comment]
Another design request: can we anchor the hubspot form so that even if the user scrolls to view all the content (see https://micromasters-ci-pr-4749.herokuapp.com/organizations/) it won't scroll out of the viewport? (we did something similar with the top nav on xPRO, I think)

**Comment by umarmughal824** [comment]
> Another design request: can we anchor the Hubspot form so that even if the user scrolls to view all the content (see https://micromasters-ci-pr-4749.herokuapp.com/organizations/) it won't scroll out of the viewport? (we did something similar with the top nav on xPRO, I think)

Let's merge it first in the home page redesign. then we will cover the design part there.

**Pull request merged**